Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageSendButton_Tests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatMessageSendButton_Tests: XCTestCase {
+    private lazy var container = UIView().withoutAutoresizingMaskConstraints
+
+    override func setUp() {
+        super.setUp()
+        container.subviews.forEach { $0.removeFromSuperview() }
+    }
+
+    func test_defaultAppearance_newMessage() {
```

Comment:
Nit: `test_defaultAppearance` I think is enough, no need to append the `_newMessage`

Attention: Keep in mind if you rename, you need to delete the previous screenshots with the old name.